person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative './nameable'
 
 class Person < Nameable
-  attr_accessor :age, :name
+  attr_accessor :age, :name, :rentals
   attr_reader :id
 
-  def initialize(_id, age, parent_permission: true, name: 'unknown')
-    super(name)
+  def initialize(age, parent_permission: true)
```

Comment:
- [ ]  In you constructor function, you should also pass it a parameter of name with a default value if `"Unknown"`, this will help you to validate your app so that when the user does not fill anything, the name will be filled with the Unknown by default.

Reply:
Thank you for the helpful review. surely I will come up with the solution